docs/aerial_autonomy_stacks.md

```diff
@@ -14,17 +14,17 @@ the following autonomy stack table was extracted and adapted.
 
 | Flight stack   |  Middleware | last  update |
 | -------------- | ----- | ------- |
-| [Aerostack(1)](https://github.com/cvar-upm/aerostack/wiki)       | ROS        | 10/2021      |
-| [Aerostack2](https://aerostack2.github.io/)       | ROS 2      | 08/2024      | 
+| [Aerostack(1)](https://github.com/cvar-upm/aerostack/wiki)       | ROS        | 11/2021 |
+| [Aerostack2](https://github.com/aerostack2/aerostack2)       | ROS 2      | 04/2025 | 
 | [Agilicious](https://agilicious.readthedocs.io/en/latest/index.html)       | ROS        | 03/2023      | 
-| [KumarRobotics Autonomy Stack](https://github.com/KumarRobotics/kr_autonomous_flight)  | ROS        | 2022     |
-| [CrazyChoir](https://github.com/OPT4SMART/crazychoir)     | ROS 2      | 09/2024    |
-| [UAL](https://github.com/grvcTeam/grvc-ual)            | ROS        | 12/2022      |
-| [XTDrone](https://github.com/robin-shaun/XTDrone)       | ROS        | 03/204      | 
-| [RotorS](https://github.com/ethz-asl/rotors_simulator)         | ROS        | 07/2021      |
-| [GAAS](https://github.com/generalized-intelligence/GAAS)           | ROS        | 10/2021      | 
-| [MRS AUV System](https://github.com/ctu-mrs/mrs_uav_system)   | ROS        | 10/2024     |
-| [Crazyswarm2](https://imrclab.github.io/crazyswarm2/)         | ROS 2        | 11/2024     |
+| [KumarRobotics Autonomy Stack](https://github.com/KumarRobotics/kr_autonomous_flight)  | ROS        | 08/2023 |
+| [CrazyChoir](https://github.com/OPT4SMART/crazychoir)     | ROS 2      | 04/2025 |
+| [UAL](https://github.com/grvcTeam/grvc-ual)            | ROS        | 12/2022 |
+| [XTDrone](https://github.com/robin-shaun/XTDrone)       | ROS        | 05/2024 | 
+| [RotorS](https://github.com/ethz-asl/rotors_simulator)         | ROS        | 07/2021 |
+| [GAAS](https://github.com/generalized-intelligence/GAAS)           | ROS        | 10/2021 | 
+| [MRS AUV System](https://github.com/ctu-mrs/mrs_uav_system)   | ROS        | 03/2025 |
+| [Crazyswarm2](https://github.com/IMRCLab/crazyswarm2)         | ROS 2        | 03/2025 |
 | [Crazyswarm(1)](https://crazyswarm.readthedocs.io/en/latest/)    | ROS        | 12/2022      | 
 
```

Comment:
Maybe add a timestamp to the last time this website was updated by the ci script. Something like

```suggestion

_Table last updated on **April 30th, 2025 at 01:00:00 am UTC**_

```